<ui version="4.0" >
 <class>ConfigDglViewClass</class>
 <widget class="QWidget" name="ConfigDglViewClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ConfigDglView</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabShape" >
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Aktuelle Ansicht</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QCheckBox" name="zeigeLanguage2" >
         <property name="text" >
          <string>Wörterbuch 2 anzeigen</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QGridLayout" >
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <property name="horizontalSpacing" >
            <number>4</number>
           </property>
           <property name="verticalSpacing" >
            <number>4</number>
           </property>
           <item row="1" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Wörterbücher:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QComboBox" name="cLexika" >
             <item>
              <property name="text" >
               <string>nebeneinander</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>untereinander</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Stichwort und Übersetzung:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QComboBox" name="cLexikaObjekte" >
             <item>
              <property name="text" >
               <string>nebeneinander</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>untereinander</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Reihenfolge:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QComboBox" name="cLB" >
             <item>
              <property name="text" >
               <string>Wörter - Übersetzung</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>Übersetzung - Wörter</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QComboBox" name="cbCaption" >
             <item>
              <property name="text" >
               <string>keine</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>kurz</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>lang</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Überschrift:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>2</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Vorschau</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>4</number>
          </property>
          <property name="leftMargin" >
           <number>4</number>
          </property>
          <property name="topMargin" >
           <number>4</number>
          </property>
          <property name="rightMargin" >
           <number>4</number>
          </property>
          <property name="bottomMargin" >
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>10</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QGridLayout" >
                <property name="leftMargin" >
                 <number>4</number>
                </property>
                <property name="topMargin" >
                 <number>4</number>
                </property>
                <property name="rightMargin" >
                 <number>4</number>
                </property>
                <property name="bottomMargin" >
                 <number>4</number>
                </property>
                <property name="horizontalSpacing" >
                 <number>4</number>
                </property>
                <property name="verticalSpacing" >
                 <number>4</number>
                </property>
                <item row="0" column="0" >
                 <widget class="QLabel" name="l1" >
                  <property name="text" >
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="QLabel" name="l2" >
                  <property name="text" >
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2" >
                 <widget class="QLabel" name="l3" >
                  <property name="text" >
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>10</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Gespeicherte Ansichten</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>4</number>
         </property>
         <item>
          <widget class="QListWidget" name="storedViews" />
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>4</number>
           </property>
           <item>
            <widget class="QPushButton" name="pbAdd" >
             <property name="text" >
              <string>erstellen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbEdit" >
             <property name="text" >
              <string>bearbeiten</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbDelete" >
             <property name="text" >
              <string>löschen</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
